### DISCUSSION TITLE
How to convert models.Identity.ID to string in kratos-client-go ?

### DISCUSSION THREAD

**Discussion by b3j0f** [ory/kratos#1227] [Q&A] [answered]
Hello,

After updating my dependency to the new version of kratos-client-go, I am not able anymore to get the id of an identity.

I tried to use the Identity.MarshalBinary function in order to get ID from binary format but I expected an easier method exist.

Regards,
b3j0f

**Comment by aeneasr** [comment, marked as answer]
`string(models.Identity.ID)`

**Reply by b3j0f** [reply]
Sorry, I haven't said that I already tried this without success with this error message :

cannot convert ID (variable of type *models.UUID) to string

**Reply by b3j0f** [reply]
Your solution succeed before my last update of the kratos-client-go

**Reply by aeneasr** [reply]
`string(*models.Identity.ID)`

**Reply by b3j0f** [reply]
Holly cr... Thank you very much 😅
